client/src/main/java/io/pravega/client/admin/StreamManager.java

```diff
@@ -238,10 +238,10 @@ static StreamManager create(ClientConfig clientConfig) {
      *
      * @param scopeName The scope of the stream.
      * @param streamName The stream name.
-     * @return stream information.
+     * @return A future representing {@link StreamInfo}.
      */
     @Beta
-    StreamInfo getStreamInfo(String scopeName, String streamName);
+    CompletableFuture<StreamInfo> fetchStreamInfo(String scopeName, String streamName);
```

Comment:
I hope we are not breaking the contract as this API is marked BETA, right?

Reply:
Yes @shshashwat. 
It's mentioned in the issue and PR's description.